### PR TITLE
Option to disable RHR 

### PR DESCRIPTION
**Pull request by flippmoke**
Adds an option to disable right-hand rule validation for Polygons and Multipolygons. 

PR includes code changes and tests and light doc changes.

### CODE REVIEW

**Review by v-hal — APPROVED** [review]
This looks good to me.

**Review by axrj — COMMENTED** [review]
I think the package-lock.json file can be removed as it's not needed here.

Also, could you update the changelog?

**Comment by moham-p** [comment]
> I think the package-lock.json file can be removed as it's not needed here.
> 
> Also, could you update the changelog?

Why there was no lock file there in the first place?

File: README.md [hunk 1]
```diff
@@ -75,6 +75,18 @@ With this option enabled, geojsonhint will produce these warnings:
...
+
+GeoJSON specification defined that linear rings must follow right-hand rule, but also says that for backward compatibility reasons parsers should not rejects polygons wiht incorrect winding order. For that kind of situations geojsonhint has an option `ignoreRightHandRule` which is `false` by default. Setting this option to `true` will cause geojsonhint to skip right-hand rule validation.
+
+```js
+geojsonhint.hint(geojsonWithIncorrectWindingOrder, {
+  ignoreRightHandRule: true
+});
+````
+
+with this options enabled, geojsonhint will not produce errors of incorrect winding order.
```
Comment (axrj) [inline README.md:88]:
```suggestion
with this option enabled, geojsonhint will not validate winding order.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,11 @@
+## 3.1.0
+
+* Adds options to disable right-hand rule validation [#2981](https://github.com/mapbox/geojsonhint/pull/87)
```
Comment (axrj) [inline CHANGELOG.md:3]:
```suggestion
* Adds options to disable right-hand rule validation [#87](https://github.com/mapbox/geojsonhint/pull/87)
```

**Pull request merged**